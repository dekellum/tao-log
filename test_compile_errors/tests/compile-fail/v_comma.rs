//! Tests of *v argument count

extern crate tao_log;
use tao_log::*;

fn zero_arguments_comma() {
    infov!(,); //~ ERROR expected identifier, found `,`
}
