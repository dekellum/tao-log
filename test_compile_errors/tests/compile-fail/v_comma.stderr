error: no rules expected the token `,`
 --> $DIR/v_comma.rs:7:12
  |
7 |     infov!(,); //~ ERROR expected identifier, found `,`
  |            ^ no rules expected this token in macro call

error[E0601]: `main` function not found in crate `$CRATE`
  |
  = note: consider adding a `main` function to `$DIR/tests/compile-fail/v_comma.rs`

For more information about this error, try `rustc --explain E0601`.
